### DISCUSSION TITLE
Inquiry about the TPC-DS load test result of Duckdb

### DISCUSSION THREAD

**Discussion by woshimajintao** [duckdb/duckdb#9422] [Q&A]
I try to load 1GB using Duckdb.But I have found every time the executing time is very different.It was from 5seconds to 15seconds.
![image](https://github.com/duckdb/duckdb/assets/48515469/d1800bb0-b86b-40da-be06-cd7d0a278a93)
![image](https://github.com/duckdb/duckdb/assets/48515469/c260f743-b240-4979-8bc7-cddfc827d978)

So how can i confirm the executing time of the load test?And what about your result of load test?

**Comment by l1t1** [comment]
os has file cache, you should restart the computer to test each run

**Reply by woshimajintao** [reply]
Perfect!
